assets/cubyz/biomes/prairie.zig.zon

```diff
@@ -0,0 +1,112 @@
+.{
+	.properties = .{
+		.dry
+	},
+	.minHeight = 40,
+	.maxHeight = 44,
+	.minRadius = 100,
+	.maxRadius = 160,
+	.roughness = 1,
+	.hills = 3,
+	
+	.music = "cubyz:sunrise",
+
+	.validPlayerSpawn = true,
+	
+	.ground_structure = .{
+		"1 cubyz:dirt",
+		"3 to 4 cubyz:limestone",
+	},
+	.structures = .{
+		.{
+			.id = "cubyz:ground_patch",
+			.block = "cubyz:dry_grass",
+			.chance = 0.33,
+			.width = 5,
+			.variation = 4,
+			.depth = 1,
+			.smoothness = 0.5,
+		},
+		.{
+			.id = "cubyz:ground_patch",
+			.block = "cubyz:grass",
+			.chance = 0.05,
+			.width = 15,
+			.variation = 7,
+			.depth = 1,
+			.smoothness = 0.5,
+		},
+		.{
+			.id = "cubyz:simple_tree",
+			.leaves = "cubyz:oak_leaves",
+			.log = "cubyz:oak_log",
+			.top = "cubyz:oak_top",
+			.chance = 0.025,
+			.type = .round,
+			.height = 2,
+			.height_variation = 3,
+		},
+		.{
+			.id = "cubyz:boulder",
+			.chance = 0.1,
+			.block = "cubyz:limestone",
+			.size = 1,
+			.size_variance = 9,
+		},
+		.{
+			.id = "cubyz:boulder",
+			.chance = 0.05,
+			.block = "cubyz:limestone",
+			.size = 2,
+			.size_variance = 8,
+		},
+		.{
+			.id = "cubyz:boulder",
+			.chance = 0.025,
+			.block = "cubyz:limestone",
+			.size = 3,
+			.size_variance = 7,
+		},
+		.{
+			.id = "cubyz:boulder",
+			.chance = 0.0125,
+			.block = "cubyz:limestone",
+			.size = 4,
+			.size_variance = 6,
```

Comment:
(not a change request)
Wow, you put quite some work into changing the random number distribution of the rock size here.
Maybe we should allow parametrizing that somehow?